worker/deps/libwebrtc/libwebrtc/modules/pacing/bitrate_prober.cc

```diff
@@ -242,16 +263,22 @@ void BitrateProber::ProbeSent(int64_t now_ms, size_t bytes) {
     if (clusters_.empty())
       probing_state_ = ProbingState::kSuspended;
 
-    // TODO: jeje
+    // TODO: Remove.
     TODO_PRINT_PROBING_STATE();
   }
 }
 
 int64_t BitrateProber::GetNextProbeTime(const ProbeCluster& cluster) {
   // RTC_CHECK_GT(cluster.pace_info.send_bitrate_bps, 0);
   // RTC_CHECK_GE(cluster.time_started_ms, 0);
-  MS_ASSERT(cluster.pace_info.send_bitrate_bps > 0, "cluster.pace_info.send_bitrate_bps must be > 0");
-  MS_ASSERT(cluster.time_started_ms > 0, "cluster.time_started_ms must be > 0");
+  if (cluster.pace_info.send_bitrate_bps <= 0) {
+    MS_ERROR("cluster.pace_info.send_bitrate_bps must be > 0");
+    return 0;
+  }
+  if (cluster.time_started_ms <= 0) {
+    MS_ERROR("cluster.time_started_ms must be > 0");
+    return 0;
+  }
```

Comment:
@jmillan I'm gonna revert these changes because those were `RTC_CHECK` so assertion is needed.